src/sass/_pattern_navigation.scss

```
@@ -144,6 +149,32 @@
   }
 }
 
+.accordian-nav-menu {
```

```suggestion
.accordion-nav-menu {
```
typo, and there is another occurrence in `accordian-nav-secondary`.

---

all accordian has been eliminated.